source/release-notes/v2.1-release-notes.rst

```diff
@@ -133,6 +134,16 @@ EL9, Ubuntu 22.04 and Ubuntu 20.04 packages
 
 See :ref:`Install Software <install-software>` for instructions on how to install OnDemand using the new EL9, Ubuntu 22.04 and Ubuntu 20.04 packages.
 
+Dex behind Apache reverse proxy by default
+..........................................
+
+By default Dex now sits behind the Apache reverse proxy.
```

Comment:
Can you add a note here stressing that Dex users are going to see a behavior change.

Maybe a note that it's a more secure default so if they decide to keep it and proxy and they've opened ports, they can now close them and so on.

Reply:
I added a warning to describe the breaking change as well as why and that sites can now close the Dex ports.

Reply:
Thanks!